transfer-proxy/test-punk.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+truffle test \
+        ./test/Punk.test.js \
+        "$@"
```

Comment:
what does it mean?

Reply:
I think It's about parameters of .sh script
no need here
exchange to --compile-all